pkg/polaris/aws/aws.go

```diff
@@ -929,3 +943,190 @@ func (a API) TrustPolicies(ctx context.Context, id IdentityFunc, features []core
 
 	return trustPolicies, nil
 }
+
+// CreateCloudCluster creates a cloud cluster in the specified account AWS account.
+func (a API) CreateCloudCluster(ctx context.Context, input aws.CreateAwsClusterInput, useLatestCdmVersion bool) (cluster CloudCluster, err error) {
```

Comment:
In this function, let's use the RSC cloud account ID instead of the AWS native ID in the errors returned. I.e. use `account.ID` instead of `account.NativeID`.